src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -190,6 +190,7 @@ class IOUAmountPage extends React.Component {
             }
             return;
         }
+        this.focusTextInput();
 
```

Comment:
NAB: Not sure what our guidelines say but I would leave the empty line after the if
```suggestion

        this.focusTextInput();
```